readthedocsext/theme/templates/projects/import_base.html

```diff
@@ -71,6 +71,21 @@ <h2 class="ui small header">{% trans "Help topics" %}</h2>
                 {% else %}
                   {{ wizard.form|crispy }}
                 {% endif %}
+
+                {% comment %}
+                Temporal approach to render a new Search (category) field to select the Organization and Team
+                where the project is going to be added
+                {% endcomment %}
+                <div class="ui horizontal category search">
+                  <div class="ui icon input">
+                    <input class="prompt"
+                           type="text"
+                           placeholder="Search organization and team...">
```

Comment:
Reminder to translate strings in templates:

```suggestion
                           placeholder="{% trans "Search organizations and teams..." %}">
```